target/scripts/startup/setup-stack.sh

```diff
@@ -1259,3 +1294,110 @@ function _setup_timezone
     return 1
   fi
 }
+
+function _setup_apply_fixes
+{
+  _log 'trace' 'Removing leftover PID files from a stop/start'
+  find /var/run/ -not -name 'supervisord.pid' -name '*.pid' -delete
+  touch /dev/shm/supervisor.sock
+
+  _log 'debug' 'Checking /var/mail permissions'
+  _chown_var_mail_if_necessary || _shutdown 'Failed to fix /var/mail permissions'
+  _log 'trace' 'Permissions in /var/mail look OK'
+}
```

Comment:
No action required, can leave as-is. This is just a review comment note.

---

These fixes are a bit different.

- Supervisor fix probably is similar to the SA and ClamAV ones, and not relevant to reload in `_setup_supervisor()`? This would be run before `daemons.stack.sh` before, but presumably could be part of `_start_daemons()`?
- `/var/mail` chown still relevant. Technically, this is specific to Dovecot AFAIK, but I'm not 100% sure.